package.json

```diff
@@ -47,9 +47,9 @@
     "lint:scss": "stylelint {demosplan/DemosPlanCoreBundle/Resources/client/scss,projects}/**/*.scss"
   },
   "dependencies": {
-    "@demos-europe/demosplan-ui": "^0.3.43",
+    "@demos-europe/demosplan-ui": "^0.3.44",
```

Comment:
with the changes from the 0.3.44 version every object in the tree list (even thought it is a branch or a leaf) gets the `sortablelist__empty` class, which causes an issue --> i think the check `isBranch` have to be added here as well
![image](https://github.com/user-attachments/assets/327806eb-3cc0-499a-a47e-92533eaf7400)

Reply:
was resolved in the dptreelistnode component